packages/config/lib/options.ts

```diff
@@ -110,7 +105,9 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
   }, {
     name: 'e2e',
     // e2e runner overrides
-    defaultValue: {},
+    defaultValue: {
+      specPattern: 'cypress/integration/**/*',
```

Comment:
This should be `cypress/e2e/**/*.cy.js` but changing this means changing all our example projects, making this PR ungodly huge, so I wil come back for this soon.